api/.pipeline/config.js

```diff
@@ -117,6 +118,7 @@ const phases = {
     changeId: deployChangeId,
     telemetryCronjobSchedule: '0 0 * * *', // Daily at midnight
     telemetryCronjobDisabled: !isStaticDeployment,
+    telemetryCronjobScriptFlags: '', // use defaults
```

Comment:
Will this not override the defaults?

Reply:
I guess it depends on what the `processDeploymentTemplate` function does internally.

Reply:
I would maybe suggest including the defaults here anyways, just to make it more obvious what values are meant to be used